indexer/ftraits.hpp

```diff
@@ -165,29 +165,30 @@ class UGC : public TraitsBase<UGC, UGCItem>
 
   static bool IsUGCAvailable(feature::TypesHolder const & types)
   {
-    UGCItem item;
-    return GetValue(types, item) ? IsUGCAvailable(item.m_mask) : false;
+    auto const opt = GetValue(types);
+    return opt ? IsUGCAvailable(opt.get().m_mask) : false;
```

Comment:
Is it possible to write:
```c++
opt->m_mask
```
?
I guess they overloaded class selector operator.

Reply:
Yes, they did. I wasn't sure which one is more accurate, because e.g. with unique_ptr we use both.

Reply:
I rewrote everything without ```get()```